src/camera/camera2d.jl

```diff
@@ -138,7 +138,7 @@ function add_zoom!(scene::SceneLike, cam::Camera2D)
         zoom = Float32(x[2])
         if zoom != 0 && ispressed(scene, zoombutton) && is_mouseinside(scene)
             pa = pixelarea(scene)[]
-            z = 1f0 + (zoom * zoomspeed)
+            z = (1f0 - zoomspeed)^zoom
```

Comment:
Ah, I also switched the direction of zooming here by changing + to -.  I did that because MakieLayout/Axis uses this direction for zooming. 
It can easily be changed by changing the sign of zoomspeed. Maybe we should note that somewhere

Reply:
This makes more sense to me than what's currently used for the zoom direction.